keylime/cloud_verifier_common.py

```diff
@@ -207,10 +209,13 @@ def prepare_v(agent):
         b64_encrypted_V = agent["b64_encrypted_V"]
         logger.debug("Re-using cached encrypted V")
     else:
+        rsa_public_key = crypto.rsa_import_pubkey(agent["public_key"])
+        if not isinstance(rsa_public_key, RSAPublicKey):
```

Comment:
I am not a fan of type hints in Python, but this assert should not be in  [rsa_import_pubkey](https://github.com/keylime/keylime/blob/b6efdf5a1695ae83b1ad271af489ebe1c0e191a2/keylime/crypto.py#L44)?

Reply:
You are right. I moved it there.

I think the type hints will help a lot...